frontend/src/bundles/common/components/icon/icon.tsx

```diff
@@ -35,4 +36,18 @@ const Icon = ({ name, color, size, className }: Properties): JSX.Element => {
     );
 };
 
-export { Icon };
+const WorkoutIcon = (): JSX.Element => {
+    return <Icon name={iconsSvgr.WORKOUT} size={'lg'} />;
+};
+
+const GoalsIcon = (): JSX.Element => {
+    return <Icon name={iconsSvgr.GOALS} size={'lg'} />;
+};
+
+export { GoalsIcon, Icon, WorkoutIcon };
+export { CalendarDaysIcon as ScheduleIcon } from '@heroicons/react/16/solid';
+export {
+    QuestionMarkCircleIcon as HelpIcon,
+    ArrowLeftStartOnRectangleIcon as LogoutIcon,
+    Squares2X2Icon as OverviewIcon,
+} from '@heroicons/react/24/outline';
```

Comment:
We already have an enum for icons called `IconComponent` Let's move it here.

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/53876827/04acf60e-fe56-454b-b000-4182c95c1901)
Like this?